src/main/java/sirius/biz/storage/layer1/S3ObjectStorageSpace.java

```diff
@@ -280,8 +281,9 @@ private boolean canCopyObject(S3ObjectStorageSpace s3ObjectStorageSpace) {
     private S3Object getS3Object(String objectKey) throws IOException {
         try {
             return store.getClient().getObject(bucketName().getName(), objectKey);
-        } catch (AmazonS3Exception exception) {
-            if (exception.getStatusCode() == HttpResponseStatus.NOT_FOUND.code()) {
+        } catch (AmazonClientException exception) {
+            if (exception instanceof AmazonS3Exception s3Exception
```

Comment:
Shouldn't we catch AmazonS3Exception and  AmazonClientException separately?

Reply:
But we than have to throw IOException in both catch statements anyways. It's probably best the way it is now.